src/components/mission-planning/ContextMenu.vue

```diff
@@ -110,6 +110,18 @@
           ></v-icon>
           <span class="text-white text-sm ml-4">{{ pathCreationButtonText }}</span>
         </v-list-item>
+        <v-divider />
+        <v-list-item class="flex items-center gap-x-2 pb-2" @click="handleSetHomePosition">
+          <v-icon
+            variant="text"
+            icon="mdi-home-map-marker"
+            rounded="full"
+            size="x-small"
+            color="white"
+            class="text-[16px]"
+          ></v-icon>
+          <span class="text-white text-sm ml-4">Set home position</span>
```

Comment:
```suggestion
          <span class="text-white text-sm ml-4">Set vehicle home position</span>
```
After today's meeting, it seems relevant to clarify the distinction between this and the default view position...